src/yz_debug.erl

```diff
@@ -0,0 +1,228 @@
+%% -------------------------------------------------------------------
```

Comment:
I have found this module useful for debugging, but it's debatable if we should include it in a release.  Maybe we should move it to our tools repo?